pkgs/leak_tracker_flutter_testing/lib/src/leak_tracking_for_tests.dart

```diff
@@ -7,24 +7,48 @@ import 'package:meta/meta.dart';
 
 void _emptyLeakHandler(Leaks leaks) {}
 
-/// Leak tracking settings and helper methods for tests.
-abstract class LeakTrackingForTests {
+/// Set of helper methods to adjust default leak tracker settings for tests.
+///
+/// Some metods modify [LeakTrackingForTests.settings], others return adjusted
+/// copy of [LeakTrackingForTests.settings].
+///
+/// Use methods, which modify [LeakTrackingForTests.settings], to
+/// change default settings for set of tests,
+/// for example for package or folder (in flutter_test_config.dart) or
+/// for library (in `setUpAll`).
+///
+/// Use methods that return adjusted [LeakTrackingForTests.settings]
+/// to customize default for an individual test:
+///
+/// ```dart
+/// testWidgets(
+///     'initialTimerDuration falls within limit',
+///     leakTracking: LeakTrackingForTests.withSkippedLeaks(allNotGCed: true),
+///     (WidgetTester tester) async {
+///       ...
+/// ```
+///
+/// Update for [LeakTrackingForTests.settings] when a test is already running
+/// will throw exception.
+abstract final class LeakTrackingForTests {
   /// Current configuration for leak tracking.
+  ///
+  /// Us used by `testWidgets` if configuration is not provided for a test.
   static LeakTrackingForTestsSettings settings =
       const LeakTrackingForTestsSettings();
 
   /// Updates [settings] to pause leak tracking.
-  static void pause() => settings = paused();
+  static void pause() => settings = ignore();
 
   /// Updates [settings] to start leak tracking.
-  static void start() => settings = started();
+  static void start() => settings = track();
```

Comment:
these need to be renamed to ignore and track

Reply:
There are two static methods: 
ignore() - returns instance of settings with paused set to true
pause - updates current settings with paused set to true

It looks meaningful for me, because first one is intended to skip just one test, while second one stops leak tracking for any test, that uses default.

We can think about other names, but we need two different names to avoid collision.

Reply:
Based on the comment thread here, https://github.com/dart-lang/leak_tracker/pull/152#discussion_r1361003657, I thought we agreed to replace `pause` and `start` with `ignore` and `track`? With all four terminologies in the mix this may be difficult for a user to understand the API. It is confusing to have 4 terms instead of just 2 that are consistent throughout.

Reply:
Okay, I see, you suggest to replace it everywhere:

SkippedLeaks (and all its members) -> IgnoredLeaks 
LeakTrackingForTestsSettings.pause -> LeakTrackingForTestsSettings.ignore
Static methods `ignore` (update settings) and `ignored` (return copy of settings with ignore = true)

Reply:
> SkippedLeaks (and all its members) -> IgnoredLeaks

This wasn't part of my original suggestion, but it could make sense to keep terminology consistent throughout this API. @goderbauer do you have an opinion on naming here?

My comment was regarding current uses of `pause` and `start`. So
`LeakTrackingForTestsSettings.pause -> LeakTrackingForTestsSettings.ignore` and 
`LeakTrackingForTestsSettings.start -> LeakTrackingForTestsSettings.track`.

Also changing the `copyWith` parameter name `paused` to `ignored` or maybe `ignoreAll` would be better. Same for the class member `paused`.

Reply:
Updated everything to 'ignore'. @goderbauer , I am still open for other suggestions. Naming is important and it is better to make it right now.